src/desdemona/lifecycles/logging.clj

```diff
@@ -20,6 +20,6 @@
 (defn add-logging
   "Adds logging output to a task's output-batch."
   [job task]
-  (if-let [entry (first (filter #(= (:onyx/name %) task) (:catalog job)))]
+  (when (seq (filter #(= (:onyx/name %) task) (:catalog job)))
```

Comment:
Two changes that make it a bit more idiomatic:

- Don't use `if` (or its cousins) if you don't have a second branch
- `seq` is the canonical way to tell if a thing is empty or not